src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    // made BOUND = 100 to generate random int which will be more than 0, because we
+    // can't have figure shapes or area lower than 0
+    public static final int BOUND = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Figure defaultFigure = new Circle(Color.WHITE.name(), 10.0);
+    private Figure randomFigure;
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        switch (figureNumber) {
+            case 0:
+                double side = random.nextInt(BOUND);
+                randomFigure = new Square(colorSupplier.getRandomColor(), side);
+                break;
+            case 1:
+                double firstSide = random.nextInt(BOUND);
+                double secondSide = random.nextInt(BOUND);
+                randomFigure = new Rectangle(colorSupplier.getRandomColor(),
+                        firstSide, secondSide);
+                break;
+            case 2:
+                double firstLeg = random.nextInt(BOUND);
+                double secondLeg = random.nextInt(BOUND);
+                randomFigure = new RightTriangle(colorSupplier.getRandomColor(),
+                        firstLeg, secondLeg);

```

Comment:
make sure geometric property can't be 0. Extract some of this logic to  a private method - creation of each figure, or generation of geometric property